lib/less/tree/import.js

```diff
@@ -127,7 +127,7 @@ Import.prototype.eval = function (context) {
         }
     }
 
-    if (this.options.inline) {
+    if (this.options.inline && !this.path.currentFileInfo.reference) {
```

Comment:
Thanks!

I guess the problem here is that the new node does not have isReferenced set to false on it?
And we can avoid it all because its not possible to reference an inlined file - so we just ignore rather than setting isReferenced on the imported node?

Is it worth a comment?